kit/cli/viper.go

```diff
@@ -65,11 +65,37 @@ func NewCommand(p *Program) *cobra.Command {
 	// This normalizes "-" to an underscore in env names.
 	viper.SetEnvKeyReplacer(strings.NewReplacer("-", "_"))
 
+	configFile := viper.GetString("CONFIG_FILE")
+	if configFile == "" {
+		// defaults to looking in same directory as program running for
+		// a file `config.yaml`
+		configFile = "config.yaml"
+	}
+	viper.SetConfigFile(configFile)
+
+	// done before we bind flags to viper keys.
+	// order of precedence (1 highest -> 3 lowest):
+	//	1. flags
+	//  2. env vars
+	//	3. config file
+	if err := initializeConfig(); err != nil {
+		panic(fmt.Sprintf("invalid config file[%s] caused panic: %s", configFile, err))
```

Comment:
the panic here is purposepul, if you provide a config, and it is janky, we're gonna explode